### DISCUSSION TITLE
Bluetooth calling and dialing control

### DISCUSSION THREAD

**Discussion by edgarwideman** [pschatzmann/ESP32-A2DP#416] [Q&A] [answered]
I stumbled across this library and am super impressed with it. Tinkered around a bit but can't find any indications that it might support Bluetooth calling. 
   In my project, I am building a car infotainment app for an Android tablet. [Open AutoDash](https://github.com/Open-Auto-Dash/openautodash) In my car (2010 Ford Fusion) I removed the built-in radio, and created a custom mount for a Samsung Tab A7 tablet in its place. The speakers are powered by a JBL audio amp behind the tablet. Currently, the amp is plugged into the tablet's 3.5 mm aux jack. But I can't have in-car, hands-free calling with that configuration. And I have searched HIGH AND LOW, and can't find ANY Bluetooth module that includes calling functions that... 
1. Is a development board module, don't want off the shelf battery powered units.
2. 5 or 12-volt continuous supply.
3. silently connects and disconnects.  Very important, I HATE the "Bluetooth connected" announcement at max volume every time.

So this brings me here. Would this library be able to interface an audio-in microphone for phone calls? As well as pick up, hangup, dial pad, mute, and so on. For my project, I would communicate with USB to the tablet then have my app send commands to and from the ESP32 and update the interface accordingly. Thanks in advance for any advice.

**Comment by pschatzmann** [comment]
https://github.com/pschatzmann/ESP32-A2DP/issues/300

**Comment by edgarwideman** [comment]
Thanks for your response. But it is almost meaningless. It simply points to another person asking a somewhat similar question which went mostly unanswered. Yes, the Hands-Free Profile is what I am looking for if I need to use direct terminology. My question, and also the #300 's question is, does this library support it or not? If so then it would be great to have a bit of direction on where to look for it as the example projects never mention it. In issue #300  it appears the user did have it in HFP and only wanted it for audio so I am thinking it does support it somewhere behind the scenes. Thanks again.

**Comment by pschatzmann** [comment]
Obviously not! 
Your about the 5th person who asks the same question!

**Reply by edgarwideman** [reply, marked as answer]
Ok thanks. And if 5 people have asked its obviously not obvious. Remember this library caters to people that know next to nothing how Bluetooth actually works behind the scene. If it says Bluetooth... Some people will just expect it to also be able to make phone calls. If you would make a small paragraph in the readme to clarify to us dumb ones you probably won't need to answer more of these questions. Chears.

**Comment by pschatzmann** [comment]
The first chapter in the README.md is now "Supported Bluetooth Protocols"
